docs/protocol.md

```diff
@@ -590,6 +591,10 @@ Value is string "host:port".
 
 Send claim message to `host:port` when response is being sent.
 
+### Transport Header `cn` -- Caller Name
+
+Value is name of the service making the call.
```

Comment:
No you cannot put this in init. For autobahn, various call reqs will have cn set to ringpop; others will have cn set to hyperbahn; or autobahn;